packages/components/src/intl.ts

```diff
@@ -0,0 +1,13 @@
+const intl = {
```

Comment:
do we need something like this? Should this decision be documented somewhere

Reply:
Usually we use the translations from `react-aria`. Would a comment in the file be enough?